### PR TITLE
Added refs to Gtest/Gmock docs from the root README.md

### PR DESCRIPTION
**Pull request by Turim**
Also we consider that the downside is now we're linked to exact files within docs subdirectory, but I don't think that they are moved often.

### CODE REVIEW

**Comment by Turim** [comment]
Related: #679 #676 #662

File: README.md [hunk 1]
```diff
@@ -15,10 +15,12 @@ also an IRC channel on OFTC (irc.oftc.net) #gtest available.  Please
 join us!
 
 **Google Mock** is an extension to Google Test for writing and using C++ mock
-classes.  See the separate [Google Mock documentation](googlemock/README.md).
+classes. 
 
-More detailed documentation for googletest (including build instructions) are
-in its interior [googletest/README.md](googletest/README.md) file.
+## Documentation ##
```
Comment (BillyDonahue) [inline README.md]: This renders with the trailing ## visible.

File: README.md [hunk 2]
```diff
@@ -15,10 +15,12 @@ also an IRC channel on OFTC (irc.oftc.net) #gtest available.  Please
...
 
 **Google Mock** is an extension to Google Test for writing and using C++ mock
-classes.  See the separate [Google Mock documentation](googlemock/README.md).
+classes. 
 
-More detailed documentation for googletest (including build instructions) are
-in its interior [googletest/README.md](googletest/README.md) file.
+## Documentation ## 
+
+  * Google Test ([Readme](googletest/README.md) [Samples](googletest/docs/Samples.md) [AdvancedGuide](googletest/docs/AdvancedGuide.md) [More](googletest/docs/Documentation.md))
```
Comment (BillyDonahue) [inline README.md]: Needs commas. Needs linebreaks.

File: README.md (hunk 1 above)
Comment (Turim) [inline README.md]: Do you mean  -with trailing space visible- ? 
As for ## : 
Cannot reproduce pasting to http://dillinger.io/
I've written ## Name ## to form a new subheader in convenience with ## Features ## , ## Platforms ## . Am I wrong?

File: README.md (hunk 2 above)
Comment (Turim) [inline README.md]: I apologize for I haven't figured out about linebreaks.
Commas - what for?

File: README.md (hunk 1 above)
Comment (Turim) [inline README.md]: My bad. I'll fix.

**Comment by Turim** [comment]
I'll reopen it within a few hours. As I believe it should contain not more than 1 commit and must be convenient with the comments above.

**Pull request closed without merging**